content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+---
+title: हेलो मिनीक्यूब
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "शुरू करते हैं"
+    weight: 10
+    post: >
+      <p>तो क्या आप तैयार हैं? एक सामान्य ऐप चलने के लिए एक साधारण कुबेरनेट्स क्लस्टर बनाएं।</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
+कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
+काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेटेस वातावरण प्रदान करता है।
+
+{{< note >}}
+यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
+स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+* मिनीक्यूब में एक नमूना एप्लीकेशन डेप्लॉय करें।
```

Comment:
```suggestion
* मिनीक्यूब में एक नमूना एप्लीकेशन स्थापित करें।
```

Reply:
Isn't स्थापित == install.
Google translate says तैनात for deploy but we are keeping it as डेप्लॉय everywhere as it's a very commonly used term in context of K8s clusters and containers.

Reply:
cc @divya-mohan0209

Reply:
Should I change it to स्थापित?

Reply:
I don't think so, given the context. You may ignore the suggestion :)

Reply:
स्थापित doesn't seems right here. IMO we can leave as it is.